modules/cert-manager-certificate-mgmt.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-creating-certificate.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-certificate-mgmt_{context}"]
+= Creating certificates for user workloads
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have installed {cert-manager-operator}.
```

Comment:
Just checking - is leaving out the "1.13.0 or later" here intentional?

Reply:
Yes :)